selfdrive/car/hyundai/interface.py

```diff
@@ -248,11 +248,7 @@ def update(self, c, can_strings):
     ret.doorOpen = not self.CS.door_all_closed
     ret.seatbeltUnlatched = not self.CS.seatbelt
 
-    # low speed steer alert hysteresis logic (only for cars with steer cut off above 10 m/s)
-    if ret.vEgo < (self.CP.minSteerSpeed + 1.) and self.CP.minSteerSpeed > 10.:
-      self.low_speed_alert = True
-    if ret.vEgo > (self.CP.minSteerSpeed + 1.):
-      self.low_speed_alert = False
+    self.low_speed_alert = ret.vEgo < self.CP.minSteerSpeed
```

Comment:
@rbiasini is this OK?

Reply:
I would keep the hysteresis, otherwise you'll have a flickering alert if you drive around (self.CP.minSteerSpeed). just make it a lot smaller. Something like:
```
    if ret.vEgo < (self.CP.minSteerSpeed + 0.2) and self.CP.minSteerSpeed > 10.:	
      self.low_speed_alert = True	
    if ret.vEgo > (self.CP.minSteerSpeed + 0.7):	
      self.low_speed_alert = False
```

Reply:
ok, it is done